indexing-service/src/main/java/io/druid/indexing/common/task/IndexTask.java

```diff
@@ -151,6 +161,8 @@ public IndexTask(
     super(
         // _not_ the version, just something uniqueish
         makeId(id, ingestionSchema),
+        String.format("index_%s", makeDataSource(ingestionSchema)),
```

Comment:
That is how I originally had it, until I found this in RealtimeIndexTask:
```  @JsonCreator
  public RealtimeIndexTask(
      @JsonProperty("id") String id,
      @JsonProperty("resource") TaskResource taskResource,
      @JsonProperty("spec") FireDepartment fireDepartment
  )
  {
    super(
        id == null ? makeTaskId(fireDepartment) : id,
        String.format("index_realtime_%s", makeDatasource(fireDepartment)),
        taskResource == null ? new TaskResource(makeTaskId(fireDepartment), 1) : taskResource,
        makeDatasource(fireDepartment)
    );
    this.spec = fireDepartment;
  }
```
I'm used to working with Realtime Indexers, so I went with that. I can switch it to just be `makeId(id, ingestionSchema)` for groupId though.

If you take a look at the inline discussion with Nishant in PR history, you can see where I previously modified AbstractFixedIntervalTask.